optuna/samplers/_tpe/sampler.py

```diff
@@ -85,6 +82,13 @@ class TPESampler(BaseSampler):
       Problems <https://doi.org/10.1145/3377930.3389817>`__
     - `Multiobjective Tree-Structured Parzen Estimator <https://doi.org/10.1613/jair.1.13188>`__
 
+    Please also check our articles:
+
+    - `Significant Speed Up of Multi-Objective TPESampler in Optuna v4.0.0
+      <https://medium.com/optuna/significant-speed-up-of-multi-objective-tpesampler-in-optuna-v4-0-0-2bacdcd1d99b>`__
```

Comment:
In reST, `_` used as a suffix is considered a single character, not two, as seen in the following example. Do you know why the existing text uses two `_` characters?

```
`Link text <link URL>`_
```

```suggestion
    - `Significant Speed Up of Multi-Objective TPESampler in Optuna v4.0.0
      <https://medium.com/optuna/significant-speed-up-of-multi-objective-tpesampler-in-optuna-v4-0-0-2bacdcd1d99b>`_
```

Reply:
Both `_` and `__` are valid but have different meanings (cf. https://github.com/optuna/optuna/issues/5513).

Reply:
I followed this issue:
- https://github.com/optuna/optuna/issues/5513

But I am feeling that I am maybe misunderstanding your question?

Reply:
I see. Thank you for letting me know!